docs/usage.md

```diff
@@ -3,130 +3,441 @@
+   - Provide details such as the record's identifier or name to access a particular record. For example:
+
+    >*Open the record with ID [ALERT-ID]*
+
+   - The assistant locates and displays the record for you.
+
+Adjust the navigation command to suit your needs, such as specifying a different module or record.
+
+### Review Navigation Commands
+
+- Review your navigation commands by verifying that they direct you to the correct module or record. Make adjustments if necessary to ensure accurate access.
+
+### Integrate Navigation into Workflow
+
+- Connect the navigation commands into your workflow to enhance efficiency. Utilize the commands to quickly move between modules and access the necessary records.
+
+Effective navigation within FortiSOAR is crucial for managing tasks and accessing information efficiently. By following these steps and utilizing the FortiAI assistant's navigation capabilities, users can streamline their workflows and improve productivity.
+
+>[!NOTE]
+>Familiarize yourself with the specific commands for navigation to optimize your use of the FortiAI assistant.
+
+## Providing Context
+
+The bot automatically loads context of an already opened record so you can initiate conversation directly around the opened records.
+
+For example, you can open an alert record and ask the bot for *Source IP* contained in the record.
+
+> *What is the source IP in this record?*
+
+>[!NOTE]
+>Ensure that context provided is relevant and up-to-date to maintain accuracy in decision-making and task execution.
+
+## Link Record
+
+Linking records within the FortiAI assistant involves correlating a record with other relevant records. This functionality helps establish connections between different pieces of information, such as correlating an indicator with an existing alert, to enhance data analysis and response actions.
+
+### Use Case Example: Correlating an Indicator
+
+A common use case is correlating an indicator with an existing alert. For example, if an alert is triggered due to a specific indicator, linking the indicator to the alert helps in understanding the relationship and impact. This correlation can aid in more accurate threat analysis and response.
+
+### Linking Records Using FortiAI Assistant
+
+1. **Initiate Linking**:
+   - Issue a command to link a record with another relevant record. For example:
+
+    >*Link indicator ID [INDICATOR-ID] to alert ID [ALERT-ID]*
+
+   - The assistant establishes the connection between the specified indicator and alert.
+
+2. **Correlate Records**:
+   - Specify the records you want to correlate. For example:
+
+    >*Correlate the newly detected IP [IP-ADDRESS] with the [INCIDENT-ID]*
+
+   - The assistant identifies and establish correlations between the specified records.
+
+3. **Review Linked Records**:
+   - Review the links and correlations to ensure they are correct and relevant. Verify that the connections between records provide meaningful insights.
+
+### Review Linked Records
```

Comment:
IMO, there is no need of this "Review Linked Records" section, please remove it

Reply:
For consistency